src/test/resources/basic-type-test/Request/BasicType.elm

```diff
@@ -0,0 +1,79 @@
+{-
+  This file was generated by endpoints-elm interpreter.
+  Do not edit this file manually.
+
+  See https://github.com/scalalandio/endpoints-elm for more information.
+-}
+
+module Request.BasicType exposing (..)
+
+import Http
+import HttpBuilder exposing (RequestBuilder)
+import Json.Decode as Decode
+import Bool.Extra
+import Maybe.Extra
+
+import Uuid exposing (..)
+
+
+unitechoGet : RequestBuilder ()
+unitechoGet  =
+  HttpBuilder.get "/unit-echo"
+    |> HttpBuilder.withExpect (Http.expectStringResponse (\_ -> Ok ()))
+    |> HttpBuilder.withTimeout 30000
+
+
+stringechoGet : String -> String -> RequestBuilder String
+stringechoGet  string =
+  HttpBuilder.get ("/string-echo/" ++ )
```

Comment:
This is invalid elm, because segment is not named; I've created #7 to handle this case properly.